ui/components/component-library/avatar-network/avatar-network.stories.tsx

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
 import {
-  Size,
-  DISPLAY,
+  Display,
   TextColor,
   BackgroundColor,
   BorderColor,
-  Color,
   AlignItems,
 } from '../../../helpers/constants/design-system';
 
 import Box from '../../ui/box/box';
```

Comment:
Can we use the TS version of the `Box` from the component library?
```suggestion
import { Box } from '..'
```